src/components/Modal.js

```diff
@@ -80,6 +78,12 @@ class Modal extends Component {
 
         if (this.props.isOpen && !this.props.isStatic) {
             if (this.props.onCancel) {
+
+                // Remove open class.
+                const container = document.body;
```

Comment:
Modal.js should not contain anything related to the dom. If you fix the logic inside `BrowserModal.onToggle` this change here is not required